test/accept.c

```diff
@@ -60,7 +60,7 @@ static void close_sock_fds(int s_fd[], int c_fd[], int nr, bool fixed)
 	close_fds(c_fd, nr);
 }
 
-static void queue_send(struct io_uring *ring, int fd)
+static void* queue_send(struct io_uring *ring, int fd)
```

Comment:
Also should be a separate thing. And a style concern, '*' goes with the name as per elsewhere in the code.